CHANGES.md

```diff
@@ -1,57 +0,0 @@
-# Changes
```

Comment:
I am planning on making a Github release for each entry in this file and @mentioning each of the tagged contributors so that all of our release notes will be with the releases here: https://github.com/yaorg/node-measured/releases.

Each of these entries will look like so:

https://github.com/yaorg/node-measured/releases/tag/v1.3.0

Reply:
- [ ] Completed